DESCRIPTION

```diff
@@ -32,6 +32,7 @@ Encoding: UTF-8
 Imports: 
     cli,
     dplyr,
+    ggplot2,
```

Comment:
Ok, this will be an unpopular opinion, but hear me out: I don't think we should add this dependency.

I'm just thinking forward to a serverless version of our app. That long loading time is actually spend on data loading - a users browser has to download all the libraries used by the app and they add up! `ggplot2` is not a small library, so if we can do without it, we probably should.

Could these be replaced with ... `plot()`?

Reply:
Done!